workspace_tools/tests.py

```diff
@@ -557,6 +557,14 @@
         "automated": True,
         #"host_test": "wait_us_auto"
     },
+    {
+        "id": "MBED_35", "description": "SPI C12832 display",
+        "source_dir": join(TEST_DIR, "mbed", "spi_C12832"),
+        "dependencies": [MBED_LIBRARIES, TEST_MBED_LIB, join(PERIPHERALS, 'C12832')],
```

Comment:
None of the libraries under "libraries\tests\peripherals" is using the definitions as per your post. In fact I was using "ADXL345" as example how to add new libraries. Fancy changing all the other libraries so there is good example how it must be done? :)